### PR TITLE
diagraph 0.2.0

### PR DESCRIPTION
**Pull request by Robotechnic**
I am submitting
- [ ] a new package
- [x] an update for a package

This new version use typst to render labels instead of graphviz. It allows to add any typst element in the node's label.

Currently, it only supports nodes's labels and we will try to support other label types but it may wont be possible.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

### CODE REVIEW

File: packages/preview/diagraph/0.2.0/lib.typ [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#import "internals.typ": render
```
Comment (istudyatuni) [inline packages/preview/diagraph/0.2.0/lib.typ:1]: There is no internals.typ

Comment (Robotechnic) [reply packages/preview/diagraph/0.2.0/lib.typ:1]: I forgot to add it to my deploy script, sorry.

**Comment by istudyatuni** [comment]
There is also an extra 0.1.2/README.MD

**Comment by Robotechnic** [comment]
OK, I removed it

**Comment by laurmaedje** [comment]
There are still changes to the 0.1.2 README in the diff. You could submit them as a separate minor fix PR, but a bit unnecessary since people will read the newer README anyway.

**Comment by Robotechnic** [comment]
I revert the README to the hold one by mistake. It's fixed

**Comment by laurmaedje** [comment]
There may be some strangeness going on because I deleted the duplicate README already a while ago: https://github.com/typst/packages/commit/ae8031bb5a38acf9b5363f34c8bc717ec7ad84a4. Git with case-sensitive / case-insensitive file systems is a bit weird.

**Pull request merged**